azerothcore/azerothcore-wotlk
# fix(Core/Formations): Implemented new creature formation flag: GROUP_AI_FLAG_ACQUIRE_NEW_TARGET_ON_EVADE.

Improved GROUP_AI_FLAG_ASSIST* flags implementation.
Fixes #13394



## Issues Addressed:

- Closes #13394
- Closes https://github.com/chromiecraft/chromiecraft/issues/4172

## Tests Performed:

- Not tested.

## How to Test the Changes:

2 chars required.
pull one with hunter
attack the rest with 2nd char
FD with hunter
The hunter mob should switch the 2nd char


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
